src/qgis_plugin_dev_tools/utils/distributions.py

```diff
@@ -33,9 +38,23 @@ def get_distribution_requirements(dist: Distribution) -> Dict[str, Distribution]
         for requirement in dist.requires or []
         if "extra ==" not in requirement
     ]
-    distributions = {
-        requirement.name: distribution(requirement.name) for requirement in requirements
-    }
+    distributions = {}
+    for requirement in requirements:
+        try:
+            distributions[requirement.name] = distribution(requirement.name)
+        except importlib_metadata.PackageNotFoundError:
+            LOGGER.info(
```

Comment:
Should this be LOGGER.error or warning?

Reply:
Well it could be warning, but since it is an error in some other package, there is not necessarily anything that could be done to prevent it. I can change it to error.